test/Utilities/VariableTemplates/test_complex_models_gpu.jl

```diff
@@ -0,0 +1,110 @@
+using Test
+using StaticArrays
+using ClimateMachine.VariableTemplates
+using CUDA
+using KernelAbstractions
+using KernelAbstractions.Extras: @unroll
+CUDA.allowscalar(false)
+# CUDA.allowscalar(true) # for CPU tests
+
```

Comment:
So the code can be run with just this file and not the runtests

```suggestion
include("complex_models.jl")
```